### ISSUE TITLE
Aborted install on Pixel 7

### ISSUE THREAD

**Issue by boggleboggle** [badabing2005/PixelFlasher#119]
I am not an expert, so can't tell you much except the error log:

extracting vendor.img (700 MB) to disk... took 3.184s
archive does not contain 'vendor.sig'
Resizing 'vendor_b'                                OKAY [  0.008s]
Sending sparse 'vendor_b' 1/3 (262116 KB)          FAILED (Error reading sparse file)
fastboot: error: Command failed

2023-11-20 19:57:20 ERROR: Encountered an error while running flash script.
theCmd: "C:\Users\Paul\AppData\Roaming\PixelFlasher\factory_images\panther-up1a.231105.003\flash-pf.bat"
Return Code: 1.
Stdout: .
Stderr: None.
Aborting ...

Can you help? I am traveling and can't use phone!

Paul

UPDATE: It was a faulty cable. I was using the original Pixel cable, but there must be an issue with it. I tried a different cable and it worked!

**Comment by badabing2005** [comment]
Thanks for the update.

**Comment by boggleboggle** [comment]
One other problem though. I now get the "your system is corrupt" message when I reboot. Everything seems to work fine, but I would like to get rid of the message if possible.

I saw your comment on another site which suggested reflashing both slots. I did this (with the rooted boot images), but I still have the message.

Would flashing the stock boot images to both slots, rebooting, and then reflashing the rooted boots help?

Any other suggestions?

Paul
